CONTRIBUTORS.md

```diff
@@ -1,3 +1,4 @@
 The following people have contributed code to this project and hold the copyright on some portions (see the commit history for details):
 * Pedro Gimeno Fortea (pgimeno) \<pgimeno@users.noreply.notabug.org\>
 * Nikola Schordinger (DeatHunter) \<Discord: DH#9367\>
+* Kurt Spencer (K.jpg) \<https://github.com/KdotJPG\>
```

Comment:
The `<>` are supposed to contain a way to contact you. :/
If you want to keep it like this it's fine tho, but people already have your GitHub using the "Contributors" tab of the repo

Reply:
Hmm good point. I'll include the change in my next PR if that's good. More straightforward to combine commits that way at this point.